src/features/favorites/components/FavoritesResults.native.test.tsx

```diff
@@ -80,13 +85,32 @@ describe('FavoritesResults component', () => {
     mockUseRemoveFavorites.mockReturnValue({
       mutate,
     } as unknown as UseMutationResult<EmptyResponse, Error, number, FavoriteMutationContext>)
-    // eslint-disable-next-line local-rules/no-react-query-provider-hoc
-    const { getByText } = render(reactQueryProviderHOC(<FavoritesResults />))
-    const container = getByText(`${paginatedFavoritesResponseSnap.nbFavorites} favoris`)
+    renderFavoritesResults()
+    const container = screen.getByText(`${paginatedFavoritesResponseSnap.nbFavorites} favoris`)
     expect(container).toBeTruthy()
-    const sortByButton = getByText('Trier')
+    const sortByButton = screen.getByText('Trier')
     expect(sortByButton).toBeTruthy()
   })
+
+  it('should show favorite list banner when the user hasnt already seen it', async () => {
+    useFeatureFlagSpy.mockReturnValueOnce(true)
+    renderFavoritesResults()
+
+    expect(await screen.findByText('Crée une liste de favoris')).toBeTruthy()
+  })
+
+  //FIXME(PC-30378): update RNTL
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('should not show favorite list banner when the user has already seen the fake door', async () => {
+    useFeatureFlagSpy.mockReturnValueOnce(true)
+
+    await storage.saveObject('has_seen_fav_list_fake_door', true)
+    renderFavoritesResults()
+
+    await waitFor(() => {
+      expect(screen.queryByText('Crée une liste de favoris')).toBeFalsy()
+    })
```

Comment:
J'ai réussi à le faire marcher en mockant la liste des favoris :
```suggestion
    useFeatureFlagSpy.mockReturnValueOnce(true)
    storage.saveObject('has_seen_fav_list_fake_door', true)
        mockUseFavorites.mockReturnValueOnce({
      data: mockData,
    } as QueryObserverSuccessResult<PaginatedFavoritesResponse>)

    renderFavoritesResults()

    await waitFor(() => {
      expect(screen.queryByText('Crée une liste de favoris')).toBeFalsy()
    })
```